x-pack/legacy/plugins/ml/server/routes/README.md

```diff
@@ -0,0 +1,16 @@
+# ML Kibana API routes
+
+This folder contains ML API routes in Kibana.
+
+Each route handler requires [apiDoc](https://github.com/apidoc/apidoc) annotations in order 
+to generate documentation.
+Besides [apidoc-markdown](https://github.com/rigwild/apidoc-markdown) package is required in order to generate the markdown.
```

Comment:
Would edit this to say `The [apidoc-markdown](https://github.com/rigwild/apidoc-markdown) package is also required in order to generate the markdown.`

Reply:
https://github.com/elastic/kibana/pull/56006/commits/fc9297f3498431f41f91440b5c9ffa664401a3c5